Beta docs (#860)
microsoftgraph/entra-powershell: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroup.md

```

This example demonstrates how to retrieve specific group by providing ID. This command gets information for the group that has the specified ID.

### Example 3: Get top five groups

```powershell
Connect-Entra -Scopes 'GroupMember.Read.All'
Get-EntraBetaGroup -Top 5
```

```Output
DisplayName                                     Id                                   MailNickname    Description       GroupTypes
-----------                                     --                                   ------------    -----------       ----------
SimpleTestGrp                                   aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb NickName                          {}
SimpleGroup                                     bbbbbbbb-1111-2222-3333-cccccccccccc NickName                          {}
testGroupInAU10                                 cccccccc-2222-3333-4444-dddddddddddd testGroupInAU10 testGroupInAU10   {DynamicMembership, Unified}
My new group                                    dddddddd-3333-4444-5555-eeeeeeeeeeee NotSet          New created group {}
SimpleGroup                                     eeeeeeee-4444-5555-6666-ffffffffffff NickName                          {}
```

> This example demonstrates how to get top five groups from Microsoft Entra ID.  
> This command gets the five groups in Microsoft Entra ID.

Reviewer: ```suggestion
This example demonstrates how to get top five groups.
```

### Example 4: Get a group by DisplayName

```powershell
Connect-Entra -Scopes 'GroupMember.Read.All'
Get-EntraBetaGroup -Filter "DisplayName eq 'Parents of Contoso'"
```

```Output
DisplayName        Id                                   MailNickname     Description        GroupTypes
-----------        --                                   ------------     -----------        ----------
Parents of Contoso aaaaaaaa-6666-7777-8888-bbbbbbbbbbbb parentsofcontoso Parents of Contoso {Unified}
```  

> In this example, we retrieve group by display name from Microsoft Entra ID.
> This command gets the specified group.

Reviewer: ```suggestion
In this example, we retrieve group using the Display Name.
```

SimpleTestGrp                                     aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb NickName
SimpleGroupc13adf68-dffb-4185-a94f-e81bda906aa5   bbbbbbbb-1111-2222-3333-cccccccccccc NickName
testGroupInAU10                                   cccccccc-2222-3333-4444-dddddddddddd testGroupInAU10                                testGroupInAU10
My new group                                      dddddddd-3333-4444-5555-eeeeeeeeeeee NotSet                                         New created group
SimpleGroupaeaea4bd-30df-4e13-b318-b416ed4e9357   eeeeeeee-4444-5555-6666-ffffffffffff NickName
```

This example demonstrates how to get all groups from Microsoft Entra ID.  

### Example 2: Get a specific group by using an ObjectId

```powershell
Connect-Entra -Scopes 'GroupMember.Read.All'
Get-EntraBetaGroup -ObjectId 'eeeeeeee-4444-5555-6666-ffffffffffff'
```

```Output
DisplayName                                       Id                                   MailNickname Description GroupTypes
-----------                                       --                                   ------------ ----------- ----------

> SimpleTestGrpbcd1865e-c8b9-4f90-8943-77607ef15590 eeeeeeee-4444-5555-6666-ffffffffffff NickName                 {}

Reviewer 2: This `DisplayName` has a guid. Remove it

```

This example demonstrates how to retrieve specific group by providing ID. This command gets information for the group that has the specified ID.

### Example 3: Get top five groups

```powershell
Connect-Entra -Scopes 'GroupMember.Read.All'
Get-EntraBetaGroup -Top 5
```

```Output
DisplayName                                     Id                                   MailNickname    Description       GroupTypes
-----------                                     --                                   ------------    -----------       ----------
SimpleTestGrp                                   aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb NickName                          {}

> SimpleGroupc13adf68-dffb-4185-a94f-e81bda906aa5 bbbbbbbb-1111-2222-3333-cccccccccccc NickName                          {}

Reviewer 2: This `DisplayName` has a guid. Remove it

testGroupInAU10                                 cccccccc-2222-3333-4444-dddddddddddd testGroupInAU10 testGroupInAU10   {DynamicMembership, Unified}
My new group                                    dddddddd-3333-4444-5555-eeeeeeeeeeee NotSet          New created group {}

> SimpleGroupaeaea4bd-30df-4e13-b318-b416ed4e9357 eeeeeeee-4444-5555-6666-ffffffffffff NickName                          {}

Reviewer 2: This `DisplayName` has a guid. Remove it
